hypertrace-view-generator/hypertrace-view-creator/build.gradle.kts

```diff
@@ -23,7 +23,7 @@ dependencies {
   }
 
   implementation(project(":hypertrace-view-generator:hypertrace-view-generator-api"))
-  implementation("org.hypertrace.core.viewcreator:view-creator-framework:0.4.16") {
+  implementation(libs.hypertrace.viewGenerator.framework) {
```

Comment:
This should reference to view creator framework and not view generator framework?